env0/resource_environment_test.go

```diff
@@ -1034,7 +1034,7 @@ func TestUnitTemplatelessEnvironmentResource(t *testing.T) {
 			terragrunt_working_directory = "%s"
 			force_destroy = true
 			vcs_commands_alias = "%s"
-			template {
+			without_template_settings {
 				repository = "%s"
 				terraform_version = "%s"
```

Comment:
probably an out of scope question but are we only allowing tf templates here no tg/pulumi etc...

Reply:
@ItamarMalka - this schema is equivalent (with some minor changes) to the template creation schema.
In other words, everything should be supported.